PolyEngine/Engine/Src/ConfigBase.hpp

```diff
@@ -5,25 +5,29 @@
 
 namespace Poly
 {
-	enum class eConfigLocation
+	/// <summary>Enum used to identify source of asset.
+	enum class eResourceSource
 	{
 		ENGINE,
 		GAME,
-		LOCAL
+		NONE,
+		USR_LOCAL,
+		_COUNT
```

Comment:
So why do we need separate directories for engine and game configuration files?
How is `USR_LOCAL` different from `ENGINE` and `GAME`? Is it a parent directory? If so why do we need an explicit enumeration for it?

Reply:
We discussed this before. USR_LOCAL is e.g.: %AppData%. Engine has it's own configuration. That is not part of the Game.

Reply:
Wait, so where do `ENGINE` and `GAME` point to?
>Engine has it's own configuration. That is not part of the Game.

Well, I'm asking what's the reasoning here.